### ISSUE TITLE
zipkin-js span name customizer

### ISSUE THREAD

**Issue by ImFlog** [openzipkin/zipkin-js#96]
Hello,
Not sure if I missed something but I think there is no possibility to customize the span name in zipkin-js.

![capture d ecran 2017-03-30 a 16 08 55](https://cloud.githubusercontent.com/assets/5982268/24586913/1f7fc5f8-17ac-11e7-8654-4f4faaa76e4d.png)
For example, in the previous screenshot, I would like to show something else than "get" for the first calls (js), rest of the app is polyglot but I managed to put what I wanted to name the spans.

Did I miss something ?
Thanks in advance

**Comment by eirslett** [comment]
It's not possible to change, because it's following the B 53 tracing spec (more or less). What you can do is add custom binary annotations to the span, then you can add whatever you want.

The UI could be improved to make it easier to show custom annotations, I guess...

**Comment by ImFlog** [comment]
Thank you for your answer.
The problem is not about an particular annotation. After looking at the code, I found that the sent span looks like this : 
`{
    "traceId": "9acbf7ffa64bb1b8",
    "id": "9acbf7ffa64bb1b8",
    "name": "get",
    "timestamp": 1491153705263968,
    "duration": 1271691,
    "annotations": [`
And I use the same kind of code that the samples gives in the README. In the cujo-js code I found this : https://github.com/openzipkin/zipkin-js/blob/master/packages/zipkin-instrumentation-cujojs-rest/src/restInterceptor.js#L27
So I am wondering where does the "get" comes from ? Looks like a bug for me.
